Makefile

```diff
@@ -175,6 +175,10 @@ addlicense: $(ADDLICENSE) ## Download addlicense locally if necessary.
 $(ADDLICENSE): $(LOCALBIN)
 	test -s $(LOCALBIN)/addlicense || GOBIN=$(LOCALBIN) go install github.com/google/addlicense@$(ADDLICENSE_VERSION)
 
+.PHONY: integration-tests
+integration-tests:
+	CGO=1 go run github.com/onsi/ginkgo/v2/ginkgo --label-filter="integration" ./...
```

Comment:
why CGO=1?

Reply:
That is correct we don't need CGO in this project as we are using `github.com/digitalocean/go-libvirt` a pure Go interface for interacting with libvirt.

Reply:
my main problem is name of variable. I don't know CGO variable and i expect it is typo for CGO_ENABLED=1|0

Reply:
You are of course correct it should be `CGO_ENABLED` - we should either set it to 0 (to be on the safe side) or remove it as the default behaviour should be to build native without C support.

Reply:
Done.